### PR TITLE
Add portfolio defined in issue #7.

### PR DESCRIPTION
**Pull request by ioumu8**
Add a new class portfolio to exchange investments information between
_Alpha_ and _Simulator_.

### CODE REVIEW

File: portfolio.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, date):
+        dict.__init__(self)
+        self.date = date
+
+    def normalize_position(self):
+        """
+        Adjust the sum of position into 100%.
+        The simulator will call this method.
+        """
+        position_sum = sum(self.values())
```
Comment (b821213) [inline portfolio.py]: What if some value is negative? If we want to allow alphas to short some instruments, we must allow negative values.
Use absolute sum to make it extendable.

File: portfolio.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        if position_sum == float(0):
+            return
```
Comment (b821213) [inline portfolio.py]: As we don't allow alphas to leave the market, I think we shouldn't just let it be.
Maybe we need to raise some Error here, such as ValueError.

File: portfolio.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+"""
+Allocate the asset into different part.
```
Comment (b821213) [inline portfolio.py]: `Allocation of the asset.`

File: portfolio.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        self.date = date
+
+    def normalize(self):
+        """
+        Normalize positions to make sure the summation of absolute values is 1.
+        The simulator will call this method.
+        """
+        position_sum = sum(abs(self.values()))
+        if position_sum == float(0):
+            print("ValueError of position on date ", self.date)
```
Comment (ioumu8) [inline portfolio.py]: Is "let it be" mean like this? 
We just raise some error information and let the error happen ?

Comment (b821213) [reply portfolio.py]: No, I mean we **can't** let it be. We must raise an Error here.

File: portfolio.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        self.date = date
+
+    def normalize(self):
+        """
+        Normalize positions to make sure the summation of absolute values is 1.
+        The simulator will call this method.
+        """
+        position_sum = sum(map(abs, self.values()))
+        if position_sum == float(0):
+            raise ValueError("zero position on %s " %(self.date))
```
Comment (b821213) [inline portfolio.py]: Please add a space between % and (.

**Pull request merged**